src/Renderer/ThreeExtended/B3dmLoader.js

```diff
@@ -104,9 +106,18 @@ B3dmLoader.prototype.parse = function parse(buffer, gltfUpAxis, textDecoder) {
                 const b3dm = { gltf, batchTable };
                 resolve(b3dm);
             };
-            this.glTFLoader.parse(buffer.slice(28 + b3dmHeader.FTJSONLength +
-                b3dmHeader.FTBinaryLength + b3dmHeader.BTJSONLength +
-                b3dmHeader.BTBinaryLength), onload);
+
+            const gltf = buffer.slice(28 + b3dmHeader.FTJSONLength +
+                    b3dmHeader.FTBinaryLength + b3dmHeader.BTJSONLength +
+                    b3dmHeader.BTBinaryLength);
+
+            const onError = (ex) => {
```

Comment:
You should read the glTF version from the header instead of reacting to the GLTFReader error. You have no guarantee that the error you catch is caused by the glTF version.

Reply:
yes, I'll change that

Reply:
Ok, PR is updated. 
can I delete the `origin/3Dtiles_glTF2` branch?
I take this opportunity to know you would have no other obsolete branches to delete? (#595)

Reply:
I don't think you need to print a warning for glTF version 1, otherwise everything seems OK.

You can delete my branch. I'll take a look at what other branches can be deleted soon.

Reply:
ok thanks